dir.props

```diff
@@ -21,6 +19,10 @@
     <CliVersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(ReleaseSuffix)</CliVersionPrefix>
 
     <CliTargetFramework>netcoreapp2.0</CliTargetFramework>
-
   </PropertyGroup>
+
+  <Import Project="build/Microsoft.DotNet.Cli.InitRepo.props" />
+  <Import Condition=" Exists('$(GitCommitInfoProps)') Or '$(GeneratingPropsFile)' != 'true' " Project="$(GitCommitInfoProps)" />
```

Comment:
This looks like an msbuild bug? The property passed on the command line is only available in build.proj and not the projects it imports.